src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -446,9 +444,9 @@ private async Task<IConfidentialClientApplication> BuildConfidentialClientApplic
         /// <param name="tenant"></param>
         private async Task<string> GetAccessTokenOnBehalfOfUserFromCacheAsync(
```

Comment:
You can see how the methods become more expressive - you now know which input params can be null and which ones can't.